.github/CODEOWNERS

```
@@ -41,9 +34,14 @@
 # PRLabel: %EngSys
 /sdk/template/           @danieljurek @weshaggard
```

```suggestion
/sdk/template/           @danieljurek @weshaggard @LarryOsterman @RickWinter 
```

I believe that the template SDK is owned by the C++ team as well as the ES team.

---

Template packages are intended to be used by SDK developers who want to write an SDK for a given service. They're a starting point for those developers and as such should be owned by the SDK team.

We in EngSys do use template packages from time to time to test various release scenarios. I'm not sure how Wes and I ended up as codeowners on here. At one point we were prototyping having multiple template packages for dependency release scenarios but solved that problem by bundling all of the vcpkg releases into a single PR by default.

---

@weshaggard can you chip-in?

---

I'm good with having engsys and the sdk team listed as owners here as this pipeline is generally a litmus test for bigger issues in the repo.